### DISCUSSION TITLE
Trendmicro-integration

### DISCUSSION THREAD

**Discussion by security-companion** [Security-Onion-Solutions/securityonion#13938] [Ideas]
Hi,
I would like to ask for integration of the following 2 integrations into security-onion.
[Trend Micro Deep Security](https://www.elastic.co/docs/current/integrations/trendmicro): Collect logs from Trend Micro Deep Security with Elastic Agent.
    [Trend Micro Vision One](https://www.elastic.co/docs/current/integrations/trend_micro_vision_one): Collect logs from Trend Micro Vision One with Elastic Agent.

Best regards
security-companion

**Comment by cm-ops** [comment]
These are coming in `2.4.120.`
